app/src/common/shared/org/mozilla/vrbrowser/ui/NavigationBarWidget.java

```diff
@@ -494,9 +494,6 @@ public void onProgressChange(GeckoSession session, int progress) {
     public void onSecurityChange(GeckoSession geckoSession, SecurityInformation securityInformation) {
         if (mURLBar != null) {
             boolean isSecure = securityInformation.isSecure;
-            if (SessionStore.DEFAULT_URL.equalsIgnoreCase(SessionStore.get().gerUriFromSession(geckoSession))) {
```

Comment:
this is because the Home URI is now served over HTTPS, so `isSecure` will already be `true`